addons/website_slides/data/mail_data.xml

```diff
@@ -90,5 +90,26 @@
             <field name="auto_delete" eval="True"/>
             <field name="user_signature" eval="False"/>
         </record>
+
+        <!--Slide channel request access to responsible-->
+        <record id="mail_template_slide_channel_join_request" model="mail.template">
+            <field name="name">Course Access Request</field>
+            <field name="model_id" ref="model_slide_channel" />
+            <field name="subject">${ctx.user.name} has requested access to ${object.name}</field>
+            <field name="email_to">${object.user_id.email}</field>
```

Comment:
You can set the from, because it is always current user (you can use user.email_formatted, grep other occurences in template, don't forget the "pipe safe")